### PR TITLE
[#174212137] Test login with ResponseSuccessJson

### PR DESCRIPTION
**Pull request by BurnedMarshal**
The `acsTest` method on success login return a ResponseSuccessJson instead a ResponsePermanentRedirect.

### CODE REVIEW

**Comment by gunzip** [comment]
lgtm so far, why draft?

**Comment by BurnedMarshal** [comment]
> lgtm so far, why draft?

Is in draft because as is the tests doesn't work.

File: src/controllers/authenticationController.ts [hunk 1]
```diff
@@ -224,6 +233,40 @@ export default class AuthenticationController {
...
+  ): Promise<
+    // tslint:disable-next-line: max-union-size
+    | IResponseErrorInternal
+    | IResponseErrorValidation
+    | IResponseErrorForbiddenNotAuthorized
+    | IResponseSuccessJson<AccessToken>
+  > {
+    const acsResponse = await this.acs(userPayload);
+    if (acsResponse.kind === "IResponsePermanentRedirect") {
+      const REDIRECT_URL = clientProfileRedirectionUrl.replace("{token}", "");
```
Comment (gunzip) [inline src/controllers/authenticationController.ts:248]: may we add a comment on what this code does and why?

Comment (BurnedMarshal) [reply src/controllers/authenticationController.ts:255]: fixed in ec700e0

**Pull request merged**